src/libraries/System.Private.CoreLib/src/System/Reflection/CustomAttributeData.cs

```diff
@@ -79,6 +87,11 @@ public override string ToString()
 
         #region Public Members
         public virtual Type AttributeType => Constructor.DeclaringType!;
+
+        // Expected to be overriden
+        public virtual ConstructorInfo Constructor => null!;
```

Comment:
This matches the existing Mono behavior. The existing CoreCLR behavior is to throw NullReferenceException in some of these.